src/BooleanParser.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class BooleanParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (!isset($value)) {
```

Comment:
Isn't `!isset($value)` always true? `parse` should never be called with `$value == null` (see https://github.com/mpscholten/request-parser/blob/master/src/AbstractValueParser.php#L17)